multiterm/README

```diff
@@ -22,45 +22,57 @@ to compile the MultiTerm plugin you will need the development files for
 the VTE library.
 
 MultiTerm is written in Vala and so requires that you have the Vala
-compiler (valac) installed as well as your existing C compiler.
+compiler (valac) installed if you're compiling from Git, as well as your
+existing C compiler. If you're compiling from a source release, the Vala
+compiler is not required as the generated sources are distributed with the
+release.
 
 
-Installation
-============
+Compiling and Installing
+========================
+
+For detailed instructions, see the README and/or INSTALL files of the
+Geany-Plugins package this plugin is part of.
+
+Note that when compiling MultiTerm, the C code that the Vala compiler
+generates triggers a few C compiler warnings in most configurations. These
+warnings are (almost always) harmless, and are outside of our control.
+There's a bug open with the Vala compiler working on fixing this:
+
+https://bugzilla.gnome.org/show_bug.cgi?id=732531
 
-For detailed instructions, see the INSTALL file of the Geany-Plugins package
-this plugin is part of.
+If you want to hide these warnings, add the appropriate `-Wno-XXX` compiler
+flags to the ``MULTITERM_CFLAGS`` environment variable when configuring the
```

Comment:
Maybe I can add the `#pragma` to ignore it and ensure that file gets included in all the generated C sources?